examples/aio-sx.yaml

```diff
@@ -164,6 +164,8 @@ spec:
   contact: info@windriver.com
   description: Virtual Box Standard System
   location: vbox
+  latitude: 45.35189954974955
+  longitude: -75.91866628453701
```

Comment:
If you're going to add values to an example model, please read the README file in the examples dir. These files are not modified directly. Rather, there are "kustomize" subdirectories under examples with segments of files, and the example models get generated by the build.

Reply:
Given Don's comment, should segment be moved to its own yaml file in another directory dedicated for this GEO location feature?

Reply:
That's certainly an option, creating a new aio-sx-geo-location.yaml file from the Makefile, which would clearly show this as an optional setting.

Reply:
Please update per Melissa's suggestion, creating a new geo-location subdir under the examples/aio-sx dir and a new aio-sx-geo-location.yaml generated by the Makefile, rather than adding it to the default. I'm surprised, actually, that this isn't added to other aio-sx example files, since you'd added it to the default config.

For example of adding new subsets, see this recent update that added two:
https://github.com/Wind-River/cloud-platform-deployment-manager/commit/f07a9bcf88d5a3f496d7992310a5b77685319eb9